### ISSUE TITLE
DC-blocker and prefiltering for the EEG channels

### ISSUE THREAD

**Issue by VladislavAksiotis** [bioelectric-interfaces/nfb#64]
As an enhancement, it would be more intuitive to apply the DC-blocker and pre-filtering only to the EEG channels, without changing the input signal of the photosensor, for example.

**Comment by ossadtchi** [comment]
Yes, this is a good idea., but to do it properly we would need to introduce channel type variable.
